src/index.js

```diff
@@ -1036,12 +1036,23 @@ function renderSpec(spec, _options) {
                   function(error, chart) {
                     var view = chart({renderer: 'svg'}).update();
                     var svgText = view.svg();
-                    var fileName = options.fileName || (md5(svgText).substring(0,7) + '.svg');
+                    
+                    var now = new Date();
+                    var str = now.getUTCFullYear().toString() + "_" +
+                            (now.getUTCMonth() + 1).toString() + "_" +
+                            now.getUTCDate() + "_" +
+                            now.getUTCHours() + "_" +
+                            now.getUTCMinutes() + "_" +
+                            now.getUTCSeconds() + "_" +
+                            now.getMiliseconds;
```

Comment:
Good idea but there is a typo here. Also, I imagine we probably don't need millisecond precision in the names anyway, some maybe just remove this.

Reply:
I fixed the typo. I'd also stick to milliseconds. Imagine you run viz(repeat(100,flip)) directly after viz(repeat(10,flip)). Both commands get executed within single second and the file would get overwritten. Maybe a better approach would be to check if the file exists already. I'll dive into that.